counterpartylib/lib/transaction.py

```diff
@@ -439,6 +440,11 @@ def construct (db, tx_info, encoding='auto',
     change_quantity = 0
     sufficient_funds = False
     final_fee = fee_per_kb
+    desired_input_count = 1
+
+    if encoding == 'multisig' and data_output and util.enabled('bytespersigop'):
+        desired_input_count = len(data_output) * 2
```

Comment:
given that `data_output = (data_array, data_value)`, do you mean `desired_input_count = len(data_output[0]) * 2` ??